### ISSUE TITLE
Not showing RIG

### ISSUE THREAD

**Issue by jperrela** [totakaro/nhos-llserver#10]
I followed the readme instructions.  The web interface does not show any rigs: 

![image](https://user-images.githubusercontent.com/65288583/115147900-72998b00-a05d-11eb-9374-f28e7a98c5fa.png)

**Comment by totakaro** [comment]
Hi @jperrela Thank u for opening this issue, lets check some things first:

1. Do you see any errors on web debug console?

![image](https://user-images.githubusercontent.com/30659361/115148032-64c71400-a023-11eb-906d-9fd643065479.png)

2. Can you see logs directly from terminal using command `tail -f /var/logs/nhos/nhm/miners/*` ?

![image](https://user-images.githubusercontent.com/30659361/115148110-bcfe1600-a023-11eb-8e1e-4d258ccc97a8.png)

**Comment by jperrela** [comment]
Hi @totakaro.

Answer to #1: Uncaught SyntaxError: Unexpected identifier [myIP]/:523
Answer to #2: Yes, everything works from terminal.

Thanks for your quick response.

**Comment by totakaro** [comment]
Hmmmmm

> Uncaught SyntaxError: Unexpected identifier [myIP]/:523

Did you change the `PORT` value to `523` ?

Can you try to open in you browser http://yourIP/miners-logs ?

You should see a plain text event source like this:

![image](https://user-images.githubusercontent.com/30659361/115148949-524ed980-a027-11eb-93df-c045fbc4ad0c.png)

**Comment by jperrela** [comment]
`Did you change the PORT value to 523 ?`
I did not. 

`Can you try to open in you browser http://yourIP/miners-logs ?`
This works

**Comment by totakaro** [comment]
Oh ok nice, I think I found the bug:

The issue is because the script is parsing something wrong and generating a faulty code:

Please, can you share your page source here?

![image](https://user-images.githubusercontent.com/30659361/115149650-a60ef200-a02a-11eb-9cbf-0fa4eceec01f.png)

**Comment by jperrela** [comment]
Here you have it:
![image](https://user-images.githubusercontent.com/65288583/115152491-95359f00-a071-11eb-86d8-7ccc78130335.png)

[source.txt](https://github.com/totakaro/nhos-llserver/files/6331688/source.txt)

**Comment by totakaro** [comment]
Hmmmmm, there is a missing chunk of code in your side 😅 

![image](https://user-images.githubusercontent.com/30659361/115243727-d74df780-a0e8-11eb-8d0b-3634ed3c9420.png)

**Comment by totakaro** [comment]
@jperrela Can u share the `server.sh` script that u tried to use and showed this error? 😄

**Comment by jperrela** [comment]
Hmm, that's weird. 
[server.txt](https://github.com/totakaro/nhos-llserver/files/6338382/server.txt)
Here you have the script. I did not modify it, it should be like the one in the 1.0.7 release.

**Comment by totakaro** [comment]
Hi @jperrela  I just checked the file you provided and everything looks exactly the same, can you make sure your file is using Unix (LF) line formating? 😅 

![image](https://user-images.githubusercontent.com/30659361/115710233-bc24f700-a337-11eb-82e8-3d70b8cd4b01.png)

![image](https://user-images.githubusercontent.com/30659361/115710424-f42c3a00-a337-11eb-857f-e68603034705.png)

**Comment by totakaro** [comment]
So the `/tmp/index.html` being generated with missing code is the current issue here 😅

**Comment by jperrela** [comment]
Hi.

Yes, it was using LF. 

I downloaded that index file to a different computer and this is what I see:
![image](https://user-images.githubusercontent.com/65288583/115846250-fd91d100-a421-11eb-8625-424a7d50b395.png)
While if I access it via the Miner's IP, It is as before. I did some restarts, but still nothing.

Anything else I could try?

**Comment by totakaro** [comment]
@jperrela  Yes, please can u try the older version v1.0.5 and check if this one works for you? 

Link here: https://github.com/totakaro/nhos-llserver/tree/v1.0.5

Release here: https://github.com/totakaro/nhos-llserver/releases/tag/v1.0.5

**Comment by jperrela** [comment]
I was using v1.0.5 smoothly before updating.

**Comment by totakaro** [comment]
@jperrela Ok, I just created a new development version v1.0.8 to try  fix this issue, please feel free to check it here: 

https://raw.githubusercontent.com/totakaro/nhos-llserver/v1.0.8/scripts.sh/server.sh

More info about changes on #11

**Comment by jperrela** [comment]
I tried locally and it works. However it does not work when connecting through an VPN to my local network. I think that was the issue. Is there a way to make it work even through the VPN?

**Comment by totakaro** [comment]
Hi @jperrela 😅 Hmmmm it should work  through regardless VPN or not VPN. Did you try the branch v1.0.8?
